src/GoogleAdWordsEventForwarder.js

```diff
@@ -53,6 +70,24 @@
 
                 try {
                     if (window.gtag && forwarderSettings.enableGtag == 'True') {
+                        const eventConsentState = getEventConsentState(event);
```

Comment:
Sorry. I assumed our build process was fixing that.

Reply:
@alexs-mparticle  - Unfortunately only our core sdk build does.  Our kit ones do not have the complexity of core, but this is a ticket we should absolutely make because I agree it's annoying.

Reply:
Ticket Created: https://go.mparticle.com/work/SQDSDKS-6166